peps/pep-0501.rst

```diff
@@ -12,6 +12,8 @@ Post-History: `08-Aug-2015 <https://mail.python.org/archives/list/python-dev@pyt
               `05-Sep-2015 <https://mail.python.org/archives/list/python-dev@python.org/thread/ILVRPS6DTFZ7IHL5HONDBB6INVXTFOZ2/>`__,
               `09-Mar-2023 <https://discuss.python.org/t/pep-501-reopen-general-purpose-string-template-literals/24625>`__,
 
+.. TODO: Start new PEP 501 d.p.o thread once these updates have been merged
```

Comment:
Note that I'm not in any hurry to actually start that thread, since I'd prefer to wait until the next iteration of PEP 750 has also been published.